Myrrhe/aston-projet-2
# feat: Added the first files for the database, and added a migration.

This branch add many files in the _symfony/src/Entity/_ folder, and a migration as well in order to set up the database.

To test it :

1. Run `php symfony/bin/console doctrine:database:create` in order to create the database if it's not already done.

> Note : Don't forget to configure your _env._ file. Check [the documentation](https://symfony.com/doc/current/doctrine.html#configuring-the-database) for more infos.

2. Run `php symfony/bin/console doctrine:migrations:migrate` to create the tables in the database.
